cves/CVE-2016-4436.yml

```diff
@@ -142,7 +148,10 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this section
     by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    the vulnerability was introduced nearly 10 years before it was discovered
+    there is a lot that happened inbetween the two commits, there are many interesting
```

Comment:
Is there anything in between the two commits that may have led the developers to go down a path that would create the vulnerability?